### ISSUE TITLE
omni_config.sample has a typo in a nickname

### ISSUE THREAD

**Issue by innnkkki** [GENI-NSF/geni-tools#158]
Luisa reported that in the omni_config.sample there is a typo in the nicknames :
```
of-rutgers=,https://foam.oflow.cip.gatech.edu:3626/foam/gapi/1
```
It should point to the foam in rutgers not in gatech. 

I checked and omni-configure.py has the same typo. 

 _Imported from trac ticket [#158](http://trac.gpolab.bbn.com/gcf/ticket/158),  created by nriga on 08-16-2012 at 15:28, last modified: 08-16-2012 at 17:01_

**Comment by luisanevers** [comment]
Also the of-gatech nick_name point to Rutgers
```
of-gatech=,https://nox.orbit-lab.org:3626/foam/gapi/1
```

_Trac comment by lnevers (github user: luisanevers) on 08-16-2012 at 15:31_

**Comment by seledwards** [comment]
Also, is it time to add rack AMs to the sample and to omni-configure. 

_Trac comment by sedwards (github user: seledwards) on 08-16-2012 at 16:22_

**Comment by luisanevers** [comment]
The following ExoGENI and InstaGENI AMs should have nick_names:
```
ExoGENI BBN SM:      https://bbn-hn.exogeni.net:11443/orca/xmlrpc
ExoGENI RENCI SM:    https://rci-hn.exogeni.net:11443/orca/xmlrpc
ExoSM:               https://geni.renci.org:11443/orca/xmlrpc
BBN ExoGENI FOAM:    https://bbn-hn.exogeni.net:3626/foam/gapi/1
RENCI ExoGENI FOAM:  https://rci-hn.exogeni.net:3626/foam/gapi/1 
InstaGENI at Utah:   http://utah.geniracks.net/protogeni/xmlrpc/am
Utah InstaGENI FOAM: https://foam.utah.geniracks.net:3626/foam/gapi/1
```

Please feel free to change the names

_Trac comment by lnevers (github user: luisanevers) on 08-16-2012 at 16:33_

**Comment by innnkkki** [comment]
All these are fixed in commit : [3ce0805a8591b5785301722a0e71f04de57808a8](https://github.com/GENI-NSF/geni-tools/commit/3ce0805a8591b5785301722a0e71f04de57808a8)

I tried to use consistent naming with what we have so far, here is what I used :
```
#Exogeni AMs include OpenFlow ExoGENI AMs
eg-gpo=,https://bbn-hn.exogeni.net:11443/orca/xmlrpc
eg-renci=,https://rci-hn.exogeni.net:11443/orca/xmlrpc
eg-sm=,https://geni.renci.org:11443/orca/xmlrpc
eg-of-gpo=,https://bbn-hn.exogeni.net:3626/foam/gapi/1
eg-of-renci=,https://rci-hn.exogeni.net:3626/foam/gapi/1 

#InstaGENI AMs, include OpenFlow InstaGENI AMs
ig-utah=,http://utah.geniracks.net/protogeni/xmlrpc/am
ig-of-utah=,https://foam.utah.geniracks.net:3626/foam/gapi/1

```

If you think these are confusing, please reopen the ticket

_Trac comment by nriga (github user: innnkkki) on 08-16-2012 at 17:01_
